docs/src/deployment.md

```diff
@@ -136,13 +136,16 @@ The options for configuring logging are limited at the moment. The MongoDB daemo
     command: --quiet
 ```
 
-Other logging settings are currently fixed. These include:
+The collector, external server, and internal server all have log level `WARNING` as default. This can be overridden by setting an environment variable to `DEBUG`, `INFO`, `WARNING`, `CRITICAL` or `ERROR`.
```

Comment:
Both the string and numeric values listed in https://docs.python.org/3/library/logging.html#levels should be possible